ci_test/common_python/tools.py

```diff
@@ -753,16 +753,16 @@ def create_tests(setup_func,
         else:
             environment = {}
 
-        if os.environ.get('USER') == 'lbannusr':
-            basepath = '/p/vast1/lbannusr'
-        else:
-            basepath = '/p/vast1/lbann'
+        # if os.environ.get('USER') == 'lbannusr':
```

Comment:
We should add a comment as to why we commented this out.

Reply:
I had the comment in there `        # Empirically the cache dir cannot be on a parallel file system`